docs/OV_Runtime_UG/ov_without_dynamic_shapes.md

```diff
@@ -0,0 +1,44 @@
+# When Dynamic Shapes API is Not Applicable  {#openvino_docs_OV_UG_NoDynamicShapes}
```

Comment:
@myshevts, please recommend the right place for this content. I think this is a part of optimization guide (an advanced one). Do we have already something similar described?